source/common/runtime/runtime_features.cc

```diff
@@ -92,6 +92,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_runtime_initialized);
 // TODO(mattklein123): Also unit test this if this sticks and this becomes the default for Apple &
 // Android.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_always_use_v6);
+// TODO(pradeepcrao) reset this to true after 2 releases (1.27)
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_enable_include_histograms);
```

Comment:
Why is a FALSE_RUNTIME_GUARD instead of a RUNTIME_GUARD?

Reply:
This disables filtering the histograms by default, so envoy users using sink predicates will not experience histograms silently not being flushed to sinks.